src/ApparentHorizons/Tags.hpp

```diff
@@ -202,6 +203,26 @@ struct NormalOneForm : db::ComputeTag {
                        const aliases::OneForm<Frame>& r_hat) noexcept;
   using argument_tags = tmpl::list<DxRadius<Frame>, Rhat<Frame>>;
 };
+/// The OneOverOneFormMargnitude is the reciprocal of the magnitude of the
+/// one-form perpendicular to the horizon
+struct OneOverOneFormMagnitude : db::SimpleTag {
+  using type = DataVector;
+};
+/// Computes the reciprocal of the magnitude of the one form perpendicular to
+/// the horizon
+template <size_t Dim, typename Frame, typename DataType>
+struct OneOverOneFormMagnitudeCompute : db::ComputeTag,
+                                        OneOverOneFormMagnitude {
+  using base = OneOverOneFormMagnitude;
+  static DataVector function(
```

Comment:
Please change this to a void function that takes as a first argument `const gsl::not_null<DataVector*> one_over_magnitude`.   We are making all compute items follow this convention